backend/src/create.rs

```diff
@@ -133,12 +134,21 @@ pub fn create_from_osm(
     boundary_wgs84: MultiPolygon,
     study_area_name: Option<String>,
     demand: Option<DemandModel>,
+    mut population_zones_wgs84: Option<Vec<PopulationZone>>,
```

Comment:
The population_zone bytes are plumbed through a lot of layers. 

I'm kind of assuming once we add other metrics that we'll maybe consolidate some of the datasets, so that hopefully we're not plumbing through 5 or 6 different parameters through the entire app.

Reply:
Oh, definitely. Maybe there's one optional demand file per study area, and one optional "everything else" file per study area, and just those two are plumbed around. I was assuming all of the prioritization data gets bundled into one blob.